int64 lenclusters
geometry_msgs/Vector3[] clusters